djstripe/utils.py

```diff
@@ -18,14 +18,10 @@ def get_supported_currency_choices(api_key):
     :param api_key: The api key associated with the account from which to pull data.
     :type api_key: str
     """
-    import stripe
```

Comment:
Please leave the import where it is and fix the commit message accordingly.

Reply:
What is the reason for importing it within this function? It is not going to be more efficient, if that is the reason for putting it here.

Python maintains a global registry of imported modules and only the namespace gets updated depending on how the module gets imported. No imported module gets re-imported unless you explicitly call `importlib.reload_module(stripe)`. So module `stripe` has already been imported and loaded long before this function or module tries to import it.

In fact importing `stripe` within this function will be slightly less efficient as the namespace binding will have to be created whenever the function gets called and destroyed when the function gets destroyed.

Moreover, in my opinion, importing modules within functions hurts importability.